source/blog/2017-10-10-glimmer-progress-report.md

```diff
@@ -0,0 +1,645 @@
+
+There were two primary motivations for releasing Glimmer.js as a standalone project:
+
+1. We wanted people who aren't "all in" on Ember to have a way to incrementally
+   adopt part of the framework.
+2. We wanted a laboratory where we could freely run experiments on what a
+   next-generation component library might look like, without creating churn for
+   Ember users.
+
+Because Ember's rendering layer is built on top of the shared Glimmer VM,
+successful experiments have a clear path to make their way upstream to Ember
+users. And, once stabilized, we'd like Glimmer.js to be the default component
+API for new Ember apps—but it's still too premature to set any timelines for
+that today.
+
+## Unlocking Experimentation
+
+While Ember is known for incorporating the best ideas from across the JavaScript
+ecosystem, it's important that we contribute back our own innovations, too.
+Refining new ideas takes time and, often, a few false starts. How do we
+reconcile the need to experiment with Ember's vaunted stability guarantees?
+
+One of the themes we discussed in our keynote was a new focus on _unlocking
+experimentation_, that is, allowing people to easily try out and share unproven
+ideas on top of the stable Ember core.
+
+Unlocking experimentation doesn't just allow for _more_ ideas; it also leads to
+_better_ ideas, because you can try things without worrying about breaking
+changes. With Glimmer.js, we wanted to eat our own experimental dogfood.
+
+We've wanted to overhaul the component API in Ember for some time now. But
+because components play such a central role, we knew that we _had_ to have a
+tangible implementation for people to play with before we could credibly ask
+them to comment on an RFC. And we knew that having an implementation would
+almost certainly shake out problems with the design that we would need to
+address.
+
+Glimmer.js is our way to iterate on a new component API until we have something
+we feel confident submitting to the Ember RFC process.
+
+We've already received incredibly useful feedback from early adopters. In
+addition, fellow core team member Chad Hietala and I have been on a team at
+LinkedIn using Glimmer.js to build a production application.
+
+[To quote DHH](http://david.heinemeierhansson.com/posts/6-why-theres-no-rails-inc):
+
+> The best frameworks are in my opinion extracted, not envisioned. And the best
+> way to extract is first to actually do.
+
+If you notice that a lot of the items I describe below are performance-related,
+that is at least partly due to our product's near-maniacal focus on mobile
+load times. We are extremely excited about some of the recent breakthroughs
+we've made and have enjoyed proving out some of the more esoteric ideas in a
+real app.
+
+## What's New in Glimmer
+
+### `<my-component />` → `<Component />`
+
+One of the most eagerly-awaited features of Glimmer.js is "angle bracket
```

Comment:
This feature isn't awaited in Glimmer.js, it is already implemented. Ember users are eagerly awaiting it for sure. Perhaps just "One of the most important experiments in Glimmer.js is"